From ff4a83ff12bbd009f38b8977d331dcedaae40350 Mon Sep 17 00:00:00 2001
From: Kuldeep Dave <kuldeep.dave@xilinx.com>
Date: Fri, 24 Aug 2018 18:36:02 +0530
Subject: [PATCH 1194/1852] PCI: XDMA PL PCIe: Enable XDMA PL PCIe for
 Microblaze

commit b5f2f46ea0ffd3ca0cba47ddf2ce333ce9dcd3fe from
https://github.com/Xilinx/linux-xlnx.git

The patch enables XDMA PL PCIe driver for Microblaze
architecture.

Signed-off-by: Kuldeep Dave <kuldeep.dave@xilinx.com>
Signed-off-by: Bharat Kumar Gogada <bharat.kumar.gogada@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/pci/controller/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/controller/Kconfig b/drivers/pci/controller/Kconfig
index 4ce4cb1563ee..af567475adb5 100644
--- a/drivers/pci/controller/Kconfig
+++ b/drivers/pci/controller/Kconfig
@@ -111,7 +111,7 @@ config PCIE_XILINX
 
 config PCIE_XDMA_PL
 	bool "Xilinx XDMA PL PCIe host bridge support"
-	depends on ARCH_ZYNQMP
+	depends on ARCH_ZYNQMP || MICROBLAZE
 	depends on PCI_MSI_IRQ_DOMAIN
 	help
 	  Say 'Y' here if you want kernel to enable support the
-- 
2.31.1

